data/presets/amenity/townhall/barangay.json

```
@@ -0,0 +1,25 @@
+{
+    "locationSet": {
+        "include": [
+            "ph"
+        ]
+    },
+    "icon": "fas-building-flag",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "barangay hall"
+    ],
+    "tags": {
+        "amenity": "townhall",
+        "townhall:type": "barangay",
+        "admin_level" : "10"
```

There's a superfluous space here:

```suggestion
        "admin_level": "10"
```